runtime/bindings/python/tests_compatibility/test_onnx/test_backend.py

```diff
@@ -221,12 +216,6 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
     (xfail_issue_38706, "OnnxBackendNodeModelTest.test_split_zero_size_splits_cpu"),
     (
         xfail_issue_38708,
-        "OnnxBackendNodeModelTest.test_slice_default_steps_cpu",
-        "OnnxBackendNodeModelTest.test_slice_negative_axes_cpu",
-        "OnnxBackendNodeModelTest.test_slice_neg_steps_cpu",
-        "OnnxBackendNodeModelTest.test_slice_neg_cpu",
-        "OnnxBackendNodeModelTest.test_slice_cpu",
-        "OnnxBackendNodeModelTest.test_slice_end_out_of_bounds_cpu",
         "OnnxBackendNodeModelTest.test_slice_start_out_of_bounds_cpu",
```

Comment:
what is the reason this test still fails?

Reply:
CPU plugin doesn't support `0` dimension in the output, I will align the xfail description in the onnx changes PR.